src/types/Instance.ts

```diff
@@ -0,0 +1,8 @@
+import { IRow } from "@patternfly/react-table";
+
+export interface Instance extends IRow {
+  id: string;
+  name: string;
+  status: string;
+  submitted_at: string;
+}
```

Comment:
Could we add short comment saying what Instance is this? I assume it is some reusable class for Process onstance, Smart-Events instance ....?

Reply:
@jomarko it is a generic `Instance`. It does not represent processor, bridge (probably it will be reused in multiple pages). As the application evolves, and more properties will come up, we will be able to specialize it or to give a particular address.

Reply:
ok @vpellegrino . thank you